src/AirtableClient.php

```diff
@@ -32,15 +31,15 @@ class AirtableClient
     protected $table = null;
 
     /**
-     * Airtable constructor. Create a new Airtable Instance
+     * Airtable constructor. Create a new Airtable Instance.
      *
      * @param string $baseId
-     * @param Table $table
+     * @param Table  $table
      */
     public function __construct(string $baseId, Table $table)
     {
         $this->client = new Client([
-            'base_uri' => $_ENV['BASE_URL'] . DIRECTORY_SEPARATOR . $_ENV['VERSION'] . DIRECTORY_SEPARATOR . $baseId . DIRECTORY_SEPARATOR
+            'base_uri' => $_ENV['BASE_URL'].DIRECTORY_SEPARATOR.$_ENV['VERSION'].DIRECTORY_SEPARATOR.$baseId.DIRECTORY_SEPARATOR,
```

Comment:
For readability, I'd rather keep the whitespace on each side of concat operators. Please correct in other places also.